### PR TITLE
Cosmos release 2024-04-26

### PR DESCRIPTION
**Pull request by FabianMeiswinkel**

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,10 @@
 ## Release History
 
-### 2.11.0-beta.1 (Unreleased)
+### 2.11.0 (2024-04-26)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:3]: we usually also mention the azure-cosmos version it upgrades to, but can do later

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:3]:
```suggestion
### 2.11.0 (2024-04-26)
#### Other Changes
* Updated `azure-cosmos` to version `4.59.0`.
```

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @FabianMeiswinkel

File: sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md [hunk 2]
```diff
@@ -26,16 +26,19 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
...
 For Spark 3.3:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12:4.29.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-3_2-12/4.29.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12:4.30.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-3_2-12/4.30.0/jar)
   
 For Spark 3.4:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12:4.29.4](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-4_2-12/4.29.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12:4.30.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-4_2-12/4.30.0/jar)
+
+For Spark 3.5:
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12:4.30.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-4_2-12/4.30.0/jar)
```
Comment (Dlougach) [inline sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md:41]: @FabianMeiswinkel The link goes to Spark 3.4 plugin.

**Pull request merged**